server/models/db/appointment.js

```diff
@@ -47,8 +47,40 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     return db.any(`update ${Tables.appointment} set status='${AppointmentStatus.cancelled}' from ${Tables.appointment} a1 inner join ${Tables.workerTimeslot} w1 on a1.worker_timeslot_id = w1.worker_timeslot_id ${queryCondition};`);
 }
 
+
+// This method updates all the worker's appointments to cancelled on a specific date.
```

Comment:
Hi Peter, 
Not sure if these comments pertain to your added function or were accidentally copied from another method. I would suggest reviewing them and discarding them if irrelevant.

Reply:
Thank you for the catch!